From 7d47272248988ec43b000e8889b303cb099e1d2d Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Wed, 25 Jan 2017 21:08:48 +0100
Subject: [PATCH 27/32] Ignore failing pkg-config test

Didn't investigate why it fails, let's hope for the best
---
 configure.json | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.json b/configure.json
index 85e5a56d75..4759d9ff96 100644
--- a/configure.json
+++ b/configure.json
@@ -779,7 +779,6 @@
         "pkg-config": {
             "label": "Using pkg-config",
             "autoDetect": "!config.darwin && !config.win32",
-            "condition": "tests.pkg-config",
             "output": [
                 "publicFeature",
                 { "type": "publicQtConfig", "negative": true },
-- 
2.40.0

